.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2.1
+jobs:
+  release:
+    docker:
+      - image: circleci/golang:1.13
+    steps:
+      - checkout
+      - run: curl -sL https://git.io/goreleaser | bash
+  build:
+    docker:
+      - image: circleci/golang:1.13
+        environment:
+          GO111MODULE: "on"
```

Comment:
Is `GO111MODULE: "on"` still necessary for go 1.13?

>The GO111MODULE environment variable continues to default to auto, but the auto setting now activates the module-aware mode of the go command whenever the current working directory contains, or is below a directory containing, a go.mod file — even if the current directory is within GOPATH/src. This change simplifies the migration of existing code within GOPATH/src and the ongoing maintenance of module-aware packages alongside non-module-aware importers.
- https://golang.org/doc/go1.13#modules

Reply:
I think I'm gonna leave it for now, but with Go 1.14 I don't mind removing it?